src/components/Menu/MenuItemPage.jsx

```diff
@@ -67,43 +66,40 @@ export default function MenuItemPage () {
       transition: {
         duration: 0.3,
         type: 'tween',
-      }
+      },
     },
     out: {
       opacity: 0,
       scale: 0.9,
-      y: "50%",
+      y: '50%',
       transition: {
         duration: 0.2,
         type: 'tween',
       },
     },
-  }
+  };
 
-  return(
+  return (
     <>
-      <OrangeNavbar needBackArrow={true}/>
-      <HeaderImage src ={currentOrder.restaurant_id === 1 ? '/Dannys_bg.png' : '/Bowl.png'}/>
+      <OrangeNavbar needBackArrow={true} />
+      <HeaderImage src={currentOrder.restaurant_id === 1 ? '/Dannys_bg.png' : '/Bowl.png'} />
       <FoodName
-        initial={{ opacity: 0}}
-        animate={{ opacity: 1}}
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
         exit={{ opacity: 0 }}
-        transition={{ duration: 0.3}}
+        transition={{ duration: 0.3 }}
       >
         {currentOrder.food}
       </FoodName>
-    <MainConatiner
-          initial="initial"
-          animate="in"
-          exit="out"
-          variants={pageVariants}
-    >
-      {/* <img src='Dannys_bg.png'/> */}
-      <StyledDescription>{item.description}</StyledDescription>
-      <p>${item.price}</p>
-      <MenuItemIncrementor/>
-      <CheckoutButton onClick={()=> clickHandler()}>Add to order{`(${item.count})`} ${totalPrice.toFixed(2)}</CheckoutButton>
-    </MainConatiner>
+      <MainConatiner initial="initial" animate="in" exit="out" variants={pageVariants}>
+        {/* <img src='Dannys_bg.png'/> */}
```

Comment:
dont think we need this might delete it lols

Reply:
delete anything you can